app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Handle signed-out notifications.
+
+define([
+  'lib/session',
+  'views/base'
+],
+function (Session, BaseView) {
+  'use strict';
+
+  return {
+    initialize: function (options) {
+      this._notifications = options.notifications;
+      this._clearSessionAndNavigateToSignIn = clearSessionAndNavigateToSignIn.bind(this);
+      this._notifications.on(
+        this._notifications.EVENTS.SIGNED_OUT,
+        this._clearSessionAndNavigateToSignIn
+      );
+    },
+
+    destroy: function () {
+      this._notifications.off(
+        this._notifications.EVENTS.SIGNED_OUT,
+        this._clearSessionAndNavigateToSignIn
+      );
+    }
+  };
+
+  function clearSessionAndNavigateToSignIn () {
+    // Unset uid otherwise it will henceforth be impossible
+    // to sign in to different accounts inside this tab.
+    this.relier.unset('uid');
+    this.relier.unset('email');
+    this.relier.unset('preVerifyToken');
+    this.user._clearSignedInAccountUid();
```

Comment:
The reason for not calling `this.user.clearSignedInAccount()` is that would broadcast the same message again, infinitely, calling out across time forever... signed out... signed out... signed out... signed out... signed out...

I'll ditch the underscore.